### PR TITLE
Reformat conv2d

### PR DESCRIPTION
**Pull request by xoiga123**

### CODE REVIEW

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by xoiga123** [comment]
@CatB1t Hi, can you review these reformats please?

File: ivy/array/layers.py [hunk 1]
```diff
@@ -445,12 +445,12 @@ def depthwise_conv2d(
     def conv2d(
         self: ivy.Array,
         filters: Union[ivy.Array, ivy.NativeArray],
-        strides: Union[int, Tuple[int], Tuple[int, int]],
+        strides: Union[int, Tuple[int, int]],
         padding: str,
         /,
         *,
         data_format: str = "NHWC",
-        dilations: Optional[Union[int, Tuple[int], Tuple[int, int]]] = 1,
```
Comment (CatB1t) [inline ivy/array/layers.py:453]: Why the optional type hint got removed?

**Review by CatB1t — CHANGES_REQUESTED** [review]
Hello @xoiga123 , could you please reference the issue you're trying to fix? we shouldn't be making changes to anything that is not related to the issue.

File: ivy/array/layers.py (hunk 1 above)
Comment (xoiga123) [reply ivy/array/layers.py:453]: Hi @CatB1t,
1. Actually it was done long ago by someone else #975, I just changed some small details, so I thought creating an issue is kinda too much.
2. I discussed this in the `function_arguments` channel in discord, and it came down like this:
> Optional should only be used in the typehints if the default value of the argument is None

**Review by CatB1t — CHANGES_REQUESTED** [review]
One final change, then we're good to merge!
- Could you please revert any changes that aren't related to `conv2d`, feel free to open a new PR to address these other functions reformatting!

Thanks!

**Comment by xoiga123** [comment]
@CatB1t Hi, I've checked and all the other stuffs that aren't related to `conv2d` is just `black` autoformatting, and they've already been merged elsewhere. I even pulled and merged from upstream with no conflict.

**Review by CatB1t — APPROVED** [review]
lgtm! ty @xoiga123 for the great work!

**Pull request merged**